lnrpc/lightning.proto

```diff
@@ -4419,6 +4419,15 @@ message PolicyUpdateRequest {
     // Optional inbound fee. If unset, the previously set value will be
     // retained [EXPERIMENTAL].
     InboundFee inbound_fee = 10;
+
+    // Under some circumstances a channel can exist with a missing edge in the
```

Comment:
Nit: => unknown circumstances.  To make it clear that this is not an expected behaviour

Reply:
Done